geps/gep-1897/index.md

```diff
@@ -238,35 +240,23 @@ Thus, the following additions would be made to the Gateway API:
-| HTTPRoute  | Listener tls               | Yes                                           | **Yes**                       |
-| HTTPRoute  | No listener tls            | Yes                                           | **Yes**                       |
-| HTTPRoute  | Listener tls               | No                                            | No                            |
-| HTTPRoute  | No listener tls            | No                                            | No                            |
-| TLSRoute   | Listener Mode: Passthrough | Yes                                           | No                            |
-| TLSRoute   | Listener Mode: Terminate   | Yes                                           | Implementation-dependent      |
-| TLSRoute   | Listener Mode: Passthrough | No                                            | No                            |
-| TLSRoute   | Listener Mode: Terminate   | No                                            | No                            |
-| TCPRoute   | Listener TLS               | Yes                                           | Implementation-dependent      |
-| TCPRoute   | No listener TLS            | Yes                                           | Implementation-dependent      |
-| TCPRoute   | Listener TLS               | No                                            | No                            |
-| TCPRoute   | No listener TLS            | No                                            | No                            |
-| UDPRoute   | Listener TLS               | Yes                                           | No                            |
-| UDPRoute   | No listener TLS            | Yes                                           | No                            |
-| UDPRoute   | Listener TLS               | No                                            | No                            |
-| UDPRoute   | No listener TLS            | No                                            | No                            |
-| GRPCRoute  | Listener TLS               | Yes                                           | Implementation-dependent      |
-| GRPCRoute  | No Listener TLS            | Yes                                           | Implementation-dependent      |
-| GRPCRoute  | Listener TLS               | No                                            | No                            |
-| GRPCRoute  | No Listener TLS            | No                                            | No                            |
+The `BackendTLSPolicy` tells a client "Connect to this service using TLS".
+This is unconditional to the type of traffic the gateway client is forwarding.
+For instance, the following will all have the gateway client add TLS:
+
+* A Gateway accepts traffic on an HTTP listener
+* A Gateway accepts and terminates TLS on an HTTPS listener
+* A Gateway accepts traffic on a TCP listener
+* A Gateway accepts traffic with `Mode: Passthrough` on a TLS listener
```

Comment:
I disagree.  Passthrough configuration should not consult the BackendTLSPolicy because there should be end-to-end encryption from client to backend for passthrough.

Reply:
@howardjohn While I think I agree with all of the other updates in this GEP, this particular one is hard for me to follow. How would this work?

Reply:
But which configuration is selected when there are both Passthrough and BackendTLSPolicy?

Reply:
I think the argument here is that both would be honored. The existing TLS connection would be passed through, and _another_ one would be originated on top of that following the instructions of BackendTLSPolicy. That would almost certainly fail, but it's with the goal of predictably following API configuration instead of trying to special case scenarios where configuration is unlikely to be valid.

With all of the above said, it's likely worth adding a note here that implementations can choose not to support BackendTLSPolicy. That decision would be per GatewayClass or Mesh, but not with individual carve outs per TLS termination mode.

Reply:
The original idea was to add a brand-new, non-existing termination "type" - backend tls termination.  You can't use passthrough with edge termination, can you? Why should we say it's permitted to use passthrough with backend termination?